cloud/deploys/staging_azure/README.md

```diff
@@ -94,11 +94,18 @@ portal the resource group name is 'tfstate' and the storage_account_name is
 Before applying the Terraform configuration, you'll need to make sure that Azure Key Vault is
 properly configured to store the `postgres-password` and `app-secret-key` secrets. To do this, run the command `az keyvault list` to list all the key vaults in your project. Find the key vault that stores the secrets for Civiform. Then run `az keyvault secret list --vault-name=[your vault name]`. The `postgres-password` and `app-secret-key` secrets should be listed.
 
-If the secrets are not listed, you'll need to [create a key vault](https://docs.microsoft.com/en-us/azure/key-vault/general/quick-create-cli) if you haven't already. Then set the secrets:
+If the secrets are not listed, you'll need to [create a key vault](https://docs.microsoft.com/en-us/azure/key-vault/general/quick-create-cli) if you haven't already. Make a note of the resource group that is used. Next, run 
```

Comment:
would add a comment about making these rbac when it asks for the security specifications

Reply:
The RBAC specification is covered in the next step where you run 
```
az keyvault update --name=[your keyvault name] --enable-rbac-authorization
```